ui/pages/Search/components/filters/CustomInheritanceFilter.jsx

```diff
@@ -55,7 +55,7 @@ const CustomInheritanceFilter = React.memo(({ value, onChange, family, individua
     <Table basic="very" compact>
       <Table.Body>
         {individuals.map((individual, i) =>
-          <Table.Row key={individual.individualGuid}>
+          <Table.Row key={individual.individualGuid} disabled={!individual.sampleGuids.length}>
```

Comment:
Simply disable the row with no data without telling the reason maybe is not user-friendly.

Reply:
fair, I'll add a hover text

Reply:
added